packages/ui/Scrollable/Scrollable.js

```diff
@@ -182,6 +182,34 @@ class ScrollableBase extends Component {
 		 */
 		onMouseDown: PropTypes.func,
 
+		/**
+		 * Called when the edges are reached.
+		 * Passes `orientation`, `position`, and `reached`.
```

Comment:
Is it correct to wrap with ' in this case?
e.g> Passes `'orientation'`, `'position'`, and `'reached'`.

Reply:
Nope. they are not string values but names of property.